packages/core/src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -345,6 +335,43 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
                 if (!paymentIntent) {
                     throw new RequestError();
                 }
+            } else if (methodId && action && action.type === 'additional_action_requires_payment_method' && action.data.token) {
+                const clientSecret = action.data.token;
+                let result;
+                let catchedConfirmError = false;
+                const stripePaymentData = this._mapStripePaymentData();
+                
+                try {
+                    result = await this._stripeUPEClient.confirmPayment(stripePaymentData);
+                } catch (error) {
```

Comment:
Is `error` used here?

Reply:
We don't use it because we don't know exactly what errors we can catch here.